score.py

```diff
@@ -209,23 +208,24 @@ def convert_nifti_to_zipped_dicom_rt(nifti_file: Path, reference_series: Path, s
     :param reference_series: Path to folder containing reference DICOM series.
     :param scratch_folder: Scratch folder to extract files into.
     :param config: Model config.
+    :param dicom_rt_zip_file_name: Target DICOM-RT zip file name, ending in .dcm.zip.
     :return: Path to DICOM-RT file.
     """
-    dicom_rt_file = scratch_folder / nifti_file.with_suffix(".dcm").name
+    dicom_rt_file_path = scratch_folder / Path(dicom_rt_zip_file_name).with_suffix("")
```

Comment:
is this with_suffix("") necessary?

Reply:
Yes, because dicom_rt_zip_file_name will be something like "segmentation.dcm.zip" and so we want the DICOM-RT file to be "segmentation.dcm"